### PR TITLE
aligns labels to fmt

### PR DESCRIPTION
**Pull request by xh3b4sd**
Some boyscouting.

### CODE REVIEW

File: service/controller/v14patch2/key/key.go [hunk 1]
```diff
@@ -64,6 +64,7 @@ const (
 	LabelApp           = "app"
 	LabelCluster       = "giantswarm.io/cluster"
 	LabelCustomer      = "customer"
+	LabelManagedBy     = "giantswarm.io/managed-by"
```
Comment (xh3b4sd) [inline service/controller/v14patch2/key/key.go]: Backporting due to e2e tests.

File: service/controller/v14patch2/resource/deployment/master_deployment.go [hunk 2]
```diff
@@ -76,10 +76,13 @@ func newMasterDeployments(customObject v1alpha1.KVMConfig) ([]*extensionsv1.Depl
 					key.VersionBundleVersionAnnotation: key.VersionBundleVersion(customObject),
 				},
 				Labels: map[string]string{
-					"cluster":  key.ClusterID(customObject),
-					"customer": key.ClusterCustomer(customObject),
-					"app":      key.MasterID,
-					"node":     masterNode.ID,
```
Comment (xh3b4sd) [inline service/controller/v14patch2/resource/deployment/master_deployment.go]: Current labels stay for now even though some are deprecated. Added new ones though.

**Review by MarcelMue — APPROVED** [review]
LGTM

**Review by rossf7 — APPROVED** [review]
<3

**Review by calvix — APPROVED** [review]

**Pull request merged**